src/snowflake/cli/_plugins/nativeapp/run_processor.py

```diff
@@ -72,20 +72,18 @@
 }
 
 
-def print_warnings(create_or_upgrade_cursor: Optional[SnowflakeCursor]):
+def print_messages(create_or_upgrade_cursor: Optional[SnowflakeCursor]):
     """
-    Shows warnings in the console returned by the CREATE or UPGRADE
+    Shows messages in the console returned by the CREATE or UPGRADE
     APPLICATION command.
     """
     if not create_or_upgrade_cursor:
         return
 
     messages = [row[0] for row in create_or_upgrade_cursor.fetchall()]
-    warnings = [m for m in messages if m.lower().startswith("warning:")]
-    if warnings:
-        for warning in warnings:
-            cc.warning(warning)
-        cc.message("")
+    for message in messages:
+        cc.warning(message)
```

Comment:
is `warning` intended here for all messages?

Reply:
For posterity: rather than try to determine what's a warning or not, we'll tolerate false positives (e.g. the `Application successfully created` message).